core/src/main/scala/sttp/tapir/util/ShadowedEndpointChecker.scala

```diff
@@ -0,0 +1,55 @@
+package sttp.tapir.util
+
+import sttp.model.Method
+import sttp.tapir.internal.RichEndpointInput
+import sttp.tapir.{Endpoint, EndpointInput, ShadowedEndpoint}
+
+import java.net.URLEncoder
+
+object ShadowedEndpointChecker {
```

Comment:
[minor] Wouldn't `FindShadowedEnpoints` be a better name? We don't check anything in the implementation of this method, but rather find the shadowed endpoints